odoo/tools/misc.py

```diff
@@ -1656,38 +1656,51 @@ def __getattr__(self, attrib):
         return DotDict(val) if type(val) is dict else val
 
 
-def get_diff(data_from, data_to, custom_style=False):
+def get_diff(data_from, data_to, custom_style=False, dark_color_scheme=False):
     """
     Return, in an HTML table, the diff between two texts.
 
     :param tuple data_from: tuple(text, name), name will be used as table header
     :param tuple data_to: tuple(text, name), name will be used as table header
     :param tuple custom_style: string, style css including <style> tag.
+    :param tuple dark_color_scheme: boolean, true if dark color scheme is used
```

Comment:
`tuple` is wrong, bad copy paste that I already [made myself](https://github.com/odoo/odoo/commit/96d3fa4e01bf8afc35dbf0b7301ce75c6bf3a5c7#diff-61162ac65633a1c7b054fc83ce1813f1a7984e3169ff36021713ef441f62a208R1482) :grimacing: 
Thinking about it, `custom_style` default value should have been `None`, but whatever.

Could you just replace
```suggestion
    :param bool dark_color_scheme: true if dark color scheme is used
```
So we don't introduce a new error?